openembedded-core-mbl/scripts/lib/mbl-signing/signing/pki/tbbr_defs.py

```diff
@@ -0,0 +1,248 @@
+# Copyright (c) 2019 Arm Limited and Contributors. All rights reserved.
```

Comment:
Can we specify that this comes from https://github.com/ARM-software/arm-trusted-firmware/blob/9d068f66b15e644df4961b74b965323c20f21f14/include/tools_share/tbbr_oid.h ?

Reply:
Probably not that exact version though possibly...

Reply:
712d58f